x-pack/test_serverless/functional/test_suites/search/common_configs/config.group4.ts

```diff
@@ -12,9 +12,9 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
 
   return {
     ...baseTestConfig.getAll(),
-    testFiles: [require.resolve('../../common/visualizations/group3')],
+    testFiles: [require.resolve('../../common/visualizations/group4')],
```

Comment:
Same as above: this needs reverting and is actually the reason for the failure you're seeing (there is no `common/visualizations/group4`).

Reply:
I think I got it right [now](https://github.com/elastic/kibana/pull/180127/commits/af5ffa3897c2eee5dbfcfa2ce19cd4f1e0c4cf52) boss.